jck/compiler.api/playlist.xml

```diff
@@ -85,7 +85,14 @@
 			<comment>Temporarily disabled on aix on JDK8 for test setup issue: backlog/issues/506</comment>
 			<platform>ppc64_aix(_mixed)?</platform>
 			<version>8</version>
+                        <impl>ibm</impl>
 		</disabled>
+                <disabled>
+                        <comment>Temporarily disabled on aix on JDK8 for test setup issue: backlog/issues/506</comment>
+                        <platform>ppc64_aix(_mixed)?</platform>
```

Comment:
All platforms default to mixed. We can remove `(_mixed)?`

Reply:
ah that's because the original code uses tabs, which when copy and pasted turns into spaces

Reply:
i'll remove the tabs

Reply:
I'll keep Mesbah's change asis

Reply:
When Mesbah added the disabled, we were using mixed. Now we do not explicitly use mixed anymore. All platforms default to mixed (except zos).

Reply:
Yes, so i'm thinking Mesbah should change those under #2713, that's not the prupose of this PR

Reply:
I am okay with the idea of cleaning up existing `(_mixed)` in a separate PR!